plugins/examples/rest-handler/build.gradle

```diff
@@ -34,21 +34,16 @@ esplugin {
 test.enabled = false
 
 tasks.register("exampleFixture", org.elasticsearch.gradle.test.AntFixture) {
-  dependsOn testClasses
-  env 'CLASSPATH', "${-> project.sourceSets.test.runtimeClasspath.asPath}"
+  dependsOn javaRestTestClasses
```

Comment:
Should be `dependsOn sourceSets.javaRestTest.runtimeClasspath`.

Reply:
done https://github.com/elastic/elasticsearch/pull/59444/commits/849d19db73f47c21baaa3cb591d28521bd9e6294